tests/beaker_tests/cisco_stp_global/test_stp_global.rb

```diff
@@ -0,0 +1,272 @@
+  manifest_props: {
+    fcoe: 'false'
+  },
+}
+
+tests[:default_plat_2] = {
+  desc:           '1.3 Default Properties platform specific part 2',
+  platform:       'n(5|6|7)k',
+  title_pattern:  'default',
+  manifest_props: {
+    domain: 'default'
+  },
+  code:           [0, 2],
+  resource:       {
+    'domain' => 'false'
+  },
+}
+
+tests[:non_default_plat_2] = {
+  desc:           '2.2 Non Default Properties platform specific part 2',
+  platform:       'n(5|6|7)k',
+  title_pattern:  'default',
+  manifest_props: {
+    domain: '100'
+  },
+}
+
+tests[:default_bd] = {
+  desc:           '1.4 bridge-domain Default Properties platform specific',
+  platform:       'n7k',
+  title_pattern:  'default',
+  manifest_props: {
+    bd_designated_priority: 'default',
+    bd_forward_time:        'default',
+    bd_hello_time:          'default',
+    bd_max_age:             'default',
+    bd_priority:            'default',
+    bd_root_priority:       'default',
+  },
+  code:           [0, 2],
+  resource:       {
+    # 'bd_designated_priority' is nil when default
+    # 'bd_forward_time' is nil when default
+    # 'bd_hello_time' is nil when default
+    # 'bd_max_age' is nil when default
+    # 'bd_priority' is nil when default
+    # 'bd_root_priority' is nil when default
+  },
+}
+
+bd_dp = Array[%w(2-42 40960), %w(83-92,100-2300 53248)]
+bd_ft = Array[%w(2-42 26), %w(83-92,100-2300 20)]
+bd_ht = Array[%w(2-42 6), %w(83-92,100-2300 9)]
+bd_ma = Array[%w(2-42 26), %w(83-92,100-2300 21)]
+bd_pri = Array[%w(2-42 40960), %w(83-92,100-2300 53248)]
+bd_rpri = Array[%w(2-42 40960), %w(83-92,100-2300 53248)]
+tests[:non_default_bd] = {
+  desc:           '2.3 bridge-domain Non Default Properties platform specific',
+  platform:       'n7k',
+  title_pattern:  'default',
+  manifest_props: {
+    bd_designated_priority: bd_dp,
+    bd_forward_time:        bd_ft,
+    bd_hello_time:          bd_ht,
+    bd_max_age:             bd_ma,
+    bd_priority:            bd_pri,
+    bd_root_priority:       bd_rpri,
+  },
+  resource:       {
+    bd_designated_priority: "#{bd_dp}",
+    bd_forward_time:        "#{bd_ft}",
+    bd_hello_time:          "#{bd_ht}",
+    bd_max_age:             "#{bd_ma}",
+    bd_priority:            "#{bd_pri}",
+    bd_root_priority:       "#{bd_rpri}",
+  },
+}
+
+#################################################################
+# TEST CASE EXECUTION
+#################################################################
+test_name "TestCase :: #{tests[:resource_name]}" do
+  # -------------------------------------------------------------------
+  device = platform
+  logger.info("#### This device is of type: #{device} #####")
+  logger.info("\n#{'-' * 60}\nSection 1. Default Property Testing")
+
+  test_harness_run(tests, :default)
+  test_harness_run(tests, :default_mst)
+  test_harness_run(tests, :default_plat_1)
+  test_harness_run(tests, :default_plat_2)
+  # test_harness_run(tests, :default_bd)
```

Comment:
As discussed in scrum, code any cleanup steps into the test so this does not need to be commented out.